services/discord.py

```diff
@@ -78,8 +78,10 @@ async def send_message(self, content: str, channel_id: int = None):
 
         If no channel ID is provided, it will use the configured bot logging channel ID.
         """
+        role_mentions = True
         if not channel_id:
             bot_logging_channel = ac.database_service.get_setting("bot_logging_channel_id")
+            role_mentions = False

```

Comment:
I'd suggest adding a `force_mentions` (or something similar) to the `send_message` function, that way if we want to ping in logging for some reason (i.e. ping admins in case of issues), we can without having to get the logging channel ID outside of the function